### ISSUE TITLE
[Bug]: Subtitle preference is not being saved

### ISSUE THREAD

**Issue by GregariousJB** [movie-web/movie-web#373]
### What happened?

According to #177 this feature was implemented on May 27th, but it was the first thing I noticed as a minor inconvenience when coming to this site for the first time the other day, so I came here to ~complain~ appeal to your mercy. Since this is officially marked as a resolved feature, I'm opening this as a bug.

### What browsers are you seeing the problem on?

Firefox

### Steps to reproduce?

1. Play the TV show Reacher - S01E04, preferably around the 17:25 mark for scientific research.
2. Bottom-right cogwheel -> Captions -> English
3. Everything is fine, but...
4. Switch to a different episode.
5. Captions are now gone again and need to be enabled again. 

Thank you for looking into this 👍

**Comment by mrjvs** [comment]
Subtitle preference saving is in the settings menu of the app, it doesn't save last selected subtitles.

Closing as it works as expected.

**Comment by GregariousJB** [comment]
Didn't know what the heck "the settings menu of the app" meant, so I went searching. Double-checked the cogwheel on the video, but nope. Found the caption language setting in the cogwheel on the homepage.

This might instead be some useful new-user feedback for those of us that would like to save subtitles and don't know where to do that. 

fmovies.wtf's subtitle saving seems simple - just set your subtitles while watching the video and it saves globally. Take them off or change them and it saves globally again. Food for thought. Thanks.
